src/screens/ENSConfirmRegisterSheet.tsx

```diff
@@ -310,8 +310,8 @@ export default function ENSConfirmRegisterSheet() {
       [REGISTRATION_STEPS.RENEW]: (
         <TransactionActionRow
           accentColor={accentColor}
-          action={() => {
-            action();
+          action={async () => {
+            await action();
```

Comment:
oh nice, having just 

`action(goToProfileScreen)` 

as other actions didn't work? that should be awaiting the callback while `executeRap`

Reply:
interesing! wasn't aware we could do that. i can change.